crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -18,9 +18,15 @@ type LoaderConfig =
       options: { [k: string]: unknown };
     };
 
-const { runLoaders } = __turbopack_external_require__(
-  "loader-runner"
-) as typeof import("loader-runner");
+let runLoaders: typeof import("loader-runner");
+try {
+  ({ runLoaders } = __turbopack_external_require__(
+    "next/dist/compiled/loader-runner"
```

Comment:
It's a bit weird to have a next.js specific import inside of `turbopack-node`...
I imagine the webpack loader executor could be general purpose.

Take a looks how the postcss transform is implemented. It also uses the next.js packaged version postcss, but that import lives on the next.js side